pywsitest/ws_response.py

```diff
@@ -33,10 +35,10 @@ def __init__(self):
         self.attributes = {}
         self.triggers = []
 
-    def __str__(self):
+    def __str__(self) -> str:
         return json.dumps(self.attributes)
 
-    def with_attribute(self, attribute, value=None):
+    def with_attribute(self, attribute, value=None) -> "WSResponse":
```

Comment:
I know this is not new but could you rename the key to attribute in the docstring please?

Reply:
Renamed, and added missing type hinting 👍